src/parameters.cpp

```diff
@@ -73,8 +77,11 @@ void Parameters::readSchemeFile(std::string conf_file_path)
         else if(str_dist(tmp,"scheme_file") <= 1){
             in >> scheme_file;
         }
-        else if(str_dist(tmp,"diffusivity") <= 1){
```

Comment:
This new case with diff_in, diff_out should not deprecate the previous versions with a single diffusivity.